merkle-patricia-tree/src/Data/Tree/MerklePatricia.hs

```diff
@@ -98,7 +103,51 @@ instance HashAlgorithm alg => Eq (MerklePatriciaTree alg a) where
 -- | An opaque 'Proof' which can be used to prove membership or atomic
 -- operations such as 'add' or 'delete'
 newtype Proof alg = Proof {unProof :: [([Bit], [(Bit, Digest alg)])]}
-  deriving stock (Show)
+  deriving stock (Eq, Show)
+
+instance HashAlgorithm alg => Serialise (Proof alg) where
```

Comment:
Do you think we could provide a `.cddl` file to document the serialization format?

Reply:
Good point! I was thinking of something like that but did not know really how to do it. I will have a look.